### ISSUE TITLE
Read Mode - Article List Issues

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PEP-Web-User-Interface#215]
On Stage site: 
   Client: 2021.01.04-5619
   Server: 8.6.2/2021.0104.2.Alpha
   DataSource: OPAS.AWSProd.2021-01-05

The Read Mode article list is no longer working.  And I don't think even when it works (sometimes) it works the way we want (which is context-sensitive).

**Issue 1.  Read mode after a search.**  
   prep: Set preferences to skip abstracts.
   a) Example: Smartsearch for:  Tuckett   (note: you must capitalize the author name to get an author search there.)
        URL: https://stage.pep-web.rocks/search?q=Tuckett
   b) Click on the first article, Basille & Tuckett.
       URL: https://stage.pep-web.rocks/read/IJP.085.0987A?q=Tuckett

so far, so good.

   c)  Backtrack (via browser) to the search URL:, https://stage.pep-web.rocks/search?q=Tuckett
   d) Click on the first article again.

The list of search results is not transferred to the left pane list this time.  URL shows as https://stage.pep-web.rocks/read/IJP.085.0987A?q=Tuckett

I also replicated this placing the Tuckett name in the regular search fields, with "Author" selected.

   https://stage.pep-web.rocks/search?searchTerms=%5B%7B"type"%3A"author"%2C"term"%3A"tuckett"%7D%5D

and with a title field search, "Heaven" (gives 15 results).

   https://stage.pep-web.rocks/search?searchTerms=%5B%7B"type"%3A"title"%2C"term"%3A"heaven"%7D%5D

**Comment by nrshapiro** [comment]
**Issue 2.  Select an article from Most Cited.**

a) Selected Klein (1946) from Most Cited info card.
    https://stage.pep-web.rocks/read/IJP.027.0099A?q=IJP.027.*

No listing of other articles from IJP.027, nor even the article I'm looking at.

**Comment by nrshapiro** [comment]
@adistasio @billdami @bakerac4 

I note the latest client has difficulty with the display of the read items on the left.  It is very inconsistent in appearing.  I presume that's just a bug.  

BUT: I note there may be confusion about how this should work...here's my summary of how it should operate.

First, Ii helps to think about Read mode being a submode rather than a top level mode.  It is a submode of Search.  It is a submode of Browse.  It should behave slightly differently in these submodes, so it may be necessary or helpful to include the calling mode in the URL., e.g., https://stage.pep-web.rocks/search/read/... and https://stage.pep-web.rocks/browse/read...  (Just a thought).

In search mode, you get the list of results in the read-list pane.  
In browse mode, you get the related issue or volume articles in the read-list pane.
Browse mode activity includes selecting items in the infocards.

Note that in Search/Read Mode selecting a reference link in any mode (from the reference section), or an item in one of the infocards, effectively puts you in Browse/Search mode and the list of articles in the read-list can be updated per browse mode rules.  The URL shall in all cases be updated, so you can backtrack...and when you do, you may end up back in /Search/Browse mode, which will again load the search set into the left read-list.

Let me know if that's not clear!

**Comment by bakerac4** [comment]
@nrshapiro I believe the only question I have come up with (so far) is which read page should the link at the top go to?

**Comment by nrshapiro** [comment]
@bakerac4 The spec calls these top of the interface menu items "pages" or "virtual pages", in other words, a application context or tab.  I think it's clearest to think of them as tabs (which is always how I presumed they would be implemented).

As a tab, the current context of that "mode" would be shown.  

TLDR: It should behave like a tab to the "Read Mode/Page"

Details:
1) Before the user has selected an article to read, this is disabled.
2) Once they have opened an article in full-text, to "Read" it (even if that was done such that they are only seeing the restricted article/abstract) it is enabled, so they can go back there.   It shows the last Read context, as a tab would.

**Comment by bakerac4** [comment]
@nrshapiro this was a larger refactor and I figured I was going to miss some items. I found some this morning - so Im re-opening this now
